__tests__/integration.test.js

```diff
@@ -107,5 +112,65 @@ describe("GET", () => {
         });
     });
   });
-  
+
+  describe("/api/articles/:article_id/comments", () => {
+    test("responds with 200 upon success", () => {
+      return request(app).get("/api/articles/1/comments").expect(200);
+    });
+
+    test("responds with array of comments from passed article_id", () => {
+      const isoDate = /^\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}.\d{3}Z$/;
+      return request(app)
+        .get("/api/articles/1/comments")
+        .then((response) => {
+          const commentsArray = response.body.articleComments;
+          expect(commentsArray).toHaveLength(11)
+          commentsArray.forEach((comment) => {
+            expect(comment).toMatchObject({
+              comment_id: expect.any(Number),
+              body: expect.any(String),
+              article_id: expect.any(Number),
+              author: expect.any(String),
+              votes: expect.any(Number),
+              created_at: expect.stringMatching(isoDate),
+            });
+          });
+        });
+    });
+
+    test("responds with 404 if article id doesn't exist", () => {
+      return request(app)
+        .get("/api/articles/999999/comments")
+        .expect(404)
+        .then((response) => {
+          expect(response.body.msg).toEqual("Not found!");
+        });
+    });
+
+    test("responds with 400 if passed article id is invalid request", () => {
+      return request(app)
+        .get("/api/articles/banana/comments")
+        .expect(400)
+        .then((response) => {
+          expect(response.body.msg).toEqual("Bad request!");
+        });
+    });
+
+    test("responds with 200 and empty array if article exists but has no comments", () => {
+      return request(app).get("/api/articles/2/comments").expect(200);
+    });
```

Comment:
It would be good to assert the response value - we'd expect an empty array and you should explicitly check for this.